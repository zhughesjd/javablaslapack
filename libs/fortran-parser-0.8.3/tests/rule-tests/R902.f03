! Test file-unit-number
!      file-unit-number  is  scalar-int-expr
! 
! Test as part of a rewind-stmt.
rewind 9
rewind x

end

