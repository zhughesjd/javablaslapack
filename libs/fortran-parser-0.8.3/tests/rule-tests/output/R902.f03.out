*******************************************
args[1]: R902.f03
R1101:main-program__begin:
R 204:specification-part: numUseStmts=0 numImportStmts=0 numDeclConstructs=0
R 406:int-literal-constant: digitString=[@10,112:112='9',<10>,5:7] kindParam=null
R 306:literal-constant:
R 701:primary:
R 902:file-unit-number:
R 925:rewind-stmt: hasPositionSpecList=false
R 214:action-stmt:
R 213:executable-construct:
R 613:part-ref: id=[@14,121:121='x',<209>,6:7] hasSelectionSubscriptList=false hasImageSelector=false
R 612:data-ref: numPartRef=1
R -3:designator-or-func-ref:
R 701:primary:
R 902:file-unit-number:
R 925:rewind-stmt: hasPositionSpecList=false
R 214:action-stmt:
R 213:executable-construct:
R 209:execution-part-construct:
R 208:execution-part:
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=true hasInternalSubprogramPart=false
Parser exiting normally
********************************************
